### PR TITLE
Cloud hosted adjustments

### PR DESCRIPTION
**Pull request by davidwyao**

### CODE REVIEW

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]
@davidwyao 
- at line 79, and 198 " to specify the location of **artists.json** on the cloud:" is not clear enough
- at line 150-154, suggest to use curl command
- at line 303, use full path of ArtistTable.js instead starting from the src directory
- at the "Building and packaging the front end" section, add "The build might take a few minutes to complete." before "You can rebuild the front..."

File: README.adoc [hunk 1]
```diff
@@ -63,13 +64,32 @@ include::{common-includes}/gitclone.adoc[]
...
 
 [role='command']
 ```
 cd finish
 mvn liberty:run
 ```
+endif::[]
+
+ifdef::cloud-hosted[]
+In this IBM cloud environment, you need to update the URL to access the **artists.json**.
```
Comment (dmuelle) [inline README.adoc:78]: use `monospace` rather than **bold** for file and directory names

File: README.adoc [hunk 2]
```diff
@@ -113,9 +163,20 @@ they are installed before the front end is built by the `frontend-maven-plugin`.
...
+Node.js is a server-side JavaScript runtime that is used for developing networking applications. Its convenient package manager, https://www.npmjs.com/[npm^], is used to run the React build scripts that are found in the `package.json` file.
+To learn more about Node.js, see the official https://nodejs.org/en/docs/[Node.js documentation^].
 
+ifndef::cloud-hosted[]
 The [hotspot=frontend-plugin]`frontend-maven-plugin` is used to [hotspot=node-resource-install]`install` the dependencies that are listed in your `package.json` file from the npm registry into a folder called `node_modules`. The `node_modules` folder can be found in your [hotspot=working-dir]`working` directory. Then, the configuration [hotspot=node-resource-build]`produces` the production files to the `src/main/frontend/build` directory. 
+endif::[]
+
+ifdef::cloud-hosted[]
+Take a look at the **pom.xml** file.
+> From the menu of the IDE, select **File** > **Open** > guide-rest-client-reactjs/start/pom.xml
```
Comment (dmuelle) [inline README.adoc:175]: see previous comment re file and directory names

File: README.adoc [hunk 3]
```diff
@@ -233,7 +295,16 @@ JSON data to allow it to be accessed by the `ReactTable`. You will notice the
 which is a relatively new sytnax made for simplicity. 
 To learn more about it, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Spread_in_object_literals[Spread in object literals^].
 
-The [hotspot=set-state]`this.setState` function is used to update the state of your React component with the data that was fetched from the server. This update triggers a rerender of your React component, which updates the table with the artist data. For more information on how state in React works, see the React documentation on https://reactjs.org/docs/faq-state.html[state and lifecycle].
+The [hotspot=set-state]`this.setState` function is used to update the state of your React component with the data that was fetched from the server. This update triggers a rerender of your React component, which updates the table with the artist data.
+For more information on how state in React works, see the React documentation on https://reactjs.org/docs/faq-state.html[state and lifecycle^].
+
+ifdef::cloud-hosted[]
+Finally, run the following command to update the URL to access the **artists.json** in the **ArtistTable.js** file:
```
Comment (dmuelle) [inline README.adoc:302]: `ArtistTable.js` file

**Review by dmuelle — CHANGES_REQUESTED** [review]
Looks good, just a few formatting issues

File: README.adoc (hunk 1 above)
Comment (gkwan-ibm) [reply README.adoc:78]: SN(cloud-hosted) cannot show monospace nicely, so we use bold

File: README.adoc (hunk 2 above)
Comment (gkwan-ibm) [reply README.adoc:175]: SN(cloud-hosted) cannot show monospace nicely, so we use bold

File: README.adoc (hunk 3 above)
Comment (gkwan-ibm) [reply README.adoc:302]: SN(cloud-hosted) cannot show monospace nicely, so we use bold

**Review by dmuelle — APPROVED** [review]
LGTM

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**